README.md

```diff
@@ -1 +1,163 @@
 # kafkaQ
+
+not sure where to put this, but add a table of contents
+
+## Overview
+
+- Monitor your Kafka application in real-time to find weaknesses and vulnerabilities
```

Comment:
weaknesses and vulnerabilities are basically the same. Why don't we say something like: "Monitor your Kafka application in real-time to find vulnerabilities and improve system health" or something like that?

Reply:
done